### PR TITLE
[SM] Change priorities of State Machine callbacks

### PR DESCRIPTION
**Pull request by arti0090**
| Q               | A
| --------------- | -----
| Branch?         | master 
| Bug fix?        | yes (potential fix)
| New feature?    | no (extending current behavior)
| BC breaks?      | possible
| Deprecations?   | no
| License         | MIT

### CODE REVIEW

File: docs/customization/state_machine.rst [hunk 1]
```diff
@@ -98,6 +103,11 @@ You will need to add these few lines to the ``config/packages/_sylius.yaml``:
                         do: ["@App\\ProductReview\\Mailer\\ConfirmationMailer", "sendEmail"]
                         # this will be the object of an Order here
                         args: ["object"]
+                        priority: 10
+
+.. tip::
+    Declared priorities will be executed in ascending order. If you are adding callback to existing state, you can find declared states and callbacks
```
Comment (GSadee) [inline docs/customization/state_machine.rst:109]:
```suggestion

    Declared priorities will be executed in ascending order. If you are adding callback to existing state, you can find declared states and callbacks
```

File: src/Sylius/Bundle/CoreBundle/Resources/config/app/state_machine/sylius_order_checkout.yml [hunk 2]
```diff
@@ -37,29 +37,34 @@ winzou_state_machine:
...
                     on: ["select_shipping"]
                     do: ["@sylius.state_resolver.order_checkout", "resolve"]
                     args: ["object"]
-                    priority: 1
+                    priority: 100
                 sylius_control_payment_state:
                     on: ["complete"]
                     do: ["@sylius.state_resolver.order_payment", "resolve"]
                     args: ["object"]
+                    priority: 0
```
Comment (GSadee) [inline src/Sylius/Bundle/CoreBundle/Resources/config/app/state_machine/sylius_order_checkout.yml:65]: As there are priorities for `complete` transition already defined:
```suggestion
                    priority: 200
```

File: src/Sylius/Bundle/CoreBundle/Resources/config/app/state_machine/sylius_order_checkout.yml [hunk 3]
```diff
@@ -37,29 +37,34 @@ winzou_state_machine:
...
                 sylius_control_shipping_state:
                     on: ["complete"]
                     do: ["@sylius.state_resolver.order_shipping", "resolve"]
                     args: ["object"]
+                    priority: 100
```
Comment (GSadee) [inline src/Sylius/Bundle/CoreBundle/Resources/config/app/state_machine/sylius_order_checkout.yml:70]:
```suggestion
                    priority: 300
```

**Review by Zales0123 — APPROVED** [review]

**Comment by SirDomin** [comment]
Thanks, @arti0090! :tada:

**Pull request merged**